### ISSUE TITLE
up to date docs? missing hi and heap_config commands

### ISSUE THREAD

**Issue by k4lizen** [pwndbg/pwndbg#2383]
The docs on https://pwndbg.readthedocs.io/en/latest/ don't have the `hi` and `heap_config` commands.

The commands are properly documented in the code from what I can see.
![image](https://github.com/user-attachments/assets/a7b9ed8c-a92c-4500-bd02-9d1d5fbdacea)

Possibly relevant: https://github.com/pwndbg/pwndbg/issues/836

**Comment by disconnect3d** [comment]
Hi, hmm I thought RTD would maybe auto update from latest dev. I have to look at it sooner or later.

**Comment by k4lizen** [comment]
The commands are still missing though

**Comment by patryk4815** [comment]
@disconnect3d ping

**Comment by disconnect3d** [comment]
@k4lizen is this still the case? can u fix it?

**Comment by k4lizen** [comment]
@disconnect3d 
Yeah it's missing on readthedocs but not on pwndbg.re . I'll look into why but we'll probably just take readthedocs down.

**Comment by disconnect3d** [comment]
We took down RTD and not use it anymore. We have https://pwndbg.re/stable/commands/glibc_ptmalloc2_heap/hi/ etc.

Closing
